### ISSUE TITLE
Need a proper build system

### ISSUE THREAD

**Issue by lap00zza** [GitHugop/AnimeScreen#6]
### Description
As we have started to use SASS, its better if we fix on a build system that everyone can use. A typical build system in NodeJS will include:
1. Gulp
2. Gulp-SASS
3. Autoprefixer

All of the above can be installed with a simple `npm install`.
What are your thoughts? Is a single build system used by everyone better? Or should we just follow what we are doing now?

**Comment by MirisWisdom** [comment]
We can recommend people to use the methods you've mentioned to fulfill the requirements (autoprefixing, sass, etc.). As long as the requirements are fulfilled, it is up to the contributor to choose whichever method they prefer regarding autoprefixing and other automated tasks. What do you think?

**Comment by lap00zza** [comment]
I personally feel its best to have a single build system that anyone can use. This has the added benefit of being able to run on a CI platform such as Travis etc. The build system can even include tests and additional task such as minification etc.

**Comment by MirisWisdom** [comment]
I agree with that point. We can make a Node build system and also write a list of requirements/steps that the build system needs to fulfill. This way, we can have a Node-based build system for everyone to use. _However_, they can also make their own system which follow the same steps as our one will, should they wish to not use Node.

Would that approach be suitable?

**Comment by lap00zza** [comment]
Sounds good to me.

**Comment by densityx** [comment]
was waiting for this 👍

**Comment by lap00zza** [comment]
I will set it up.
